### PR TITLE
[4.13-4.14] refine ec/stable ci jobs

### PR DESCRIPTION
**Pull request by jianlinliu**
Per our agreement in the team, for stable builds, we only run some important install configurations.
1. For 4.14, remove unnecessary ones.
2. For 4.13, add missing ones.

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 1]
```diff
@@ -33913,13 +33913,7 @@ periodics:
     ci-operator.openshift.io/variant: amd64-stable
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-aws-ocm-osd-ccs-p2-f7
-  reporter_config:
-    slack:
-      channel: '#ocm-qe-prow-ci-jobs'
-      job_states_to_report:
-      - failure
-      report_template: Job {{.Spec.Job}} failed. <{{.Status.URL}}|View logs>
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:33922]: We lost slack notification for this job ?

Comment (jianlinliu) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:33922]: Good catch, fixed it.

**Pull request merged**